supervisord.conf

```diff
@@ -3,34 +3,33 @@ nodaemon = true
 
 [program:nginx]
 command = /nginx.sh
-stopsignal = QUIT
 autostart = true
 autorestart = true
 stdout_logfile = /var/log/digitalmarketplace/nginx_access.log
 stdout_logfile_maxbytes = 50000000
 stdout_logfile_backups = 3
 stderr_logfile = /dev/stderr
 stderr_logfile_maxbytes = 0
+stopsignal = INT
 
 [program:uwsgi]
-command = uwsgi --ini /etc/uwsgi.conf --hook-master-start "unix_signal:10 gracefully_kill_them_all" --master
```

Comment:
If there's a chance that we can get uwsgi to respond gracefully to the SIGTERM from PaaS then should we leave this on?

Reply:
I'd rather start from scratch with any new experimentation - this PR is mostly so we can clear the release pipeline with the old boring option.

Reply:
Okay